src/Microsoft.ML.Data/Evaluators/Metrics/RankerMetrics.cs

```diff
@@ -0,0 +1,44 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+
+namespace Microsoft.ML.Data
+{
+    public sealed class RankerMetrics
+    {
+        /// <summary>
+        /// Array of normalized discounted cumulative gains where i-th element represent NDCG@i (you can also phrase it as NDCG at i).
```

Comment:
>(you can also phrase it as NDCG at i) [](start = 96, length = 37)

You don't have to write this. I just try to give you option how you can phrase it :)
it's either NDCG@i or NGDCG at i, both a valid names.